packages/commercetools/composables/src/useCheckout/shared.ts

```diff
@@ -0,0 +1,28 @@
+import { ref, Ref, computed } from '@vue/composition-api';
+import { ShippingMethod, Customer } from '@vue-storefront/commercetools-api/lib/types/GraphQL';
+
+export const initialDetails = { contactInfo: {} };
+export const paymentMethods: Ref<any[]> = ref([]);
+export const shippingMethods: Ref<any[]> = ref([]);
```

Comment:
Why `any` (here and below, on billing/shipping details)? Don't we have type from CT for those?